spec/toml-spec.coffee

```diff
@@ -18,53 +18,80 @@ describe "TOML grammar", ->
 
   it "tokenizes strings", ->
     {tokens} = grammar.tokenizeLine('"I am a string"')
-    expect(tokens[0]).toEqual value: '"', scopes: ["source.toml", "string.toml", "string.begin.toml"]
-    expect(tokens[1]).toEqual value: 'I am a string', scopes: ["source.toml", "string.toml"]
-    expect(tokens[2]).toEqual value: '"', scopes: ["source.toml", "string.toml", "string.end.toml"]
+    expect(tokens[0]).toEqual value: '"', scopes: ["source.toml", "string.quoted.double.toml", "punctuation.definition.string.begin.toml"]
+    expect(tokens[1]).toEqual value: 'I am a string', scopes: ["source.toml", "string.quoted.double.toml"]
+    expect(tokens[2]).toEqual value: '"', scopes: ["source.toml", "string.quoted.double.toml", "punctuation.definition.string.end.toml"]
 
     {tokens} = grammar.tokenizeLine('"I\'m \\n escaped"')
-    expect(tokens[0]).toEqual value: '"', scopes: ["source.toml", "string.toml", "string.begin.toml"]
-    expect(tokens[1]).toEqual value: "I'm ", scopes: ["source.toml", "string.toml"]
-    expect(tokens[2]).toEqual value: "\\n", scopes: ["source.toml", "string.toml", "constant.character.escape.toml"]
-    expect(tokens[3]).toEqual value: " escaped", scopes: ["source.toml", "string.toml"]
-    expect(tokens[4]).toEqual value: '"', scopes: ["source.toml", "string.toml", "string.end.toml"]
+    expect(tokens[0]).toEqual value: '"', scopes: ["source.toml", "string.quoted.double.toml", "punctuation.definition.string.begin.toml"]
+    expect(tokens[1]).toEqual value: "I'm ", scopes: ["source.toml", "string.quoted.double.toml"]
+    expect(tokens[2]).toEqual value: "\\n", scopes: ["source.toml", "string.quoted.double.toml", "constant.character.escape.toml"]
+    expect(tokens[3]).toEqual value: " escaped", scopes: ["source.toml", "string.quoted.double.toml"]
+    expect(tokens[4]).toEqual value: '"', scopes: ["source.toml", "string.quoted.double.toml", "punctuation.definition.string.end.toml"]
+
+    {tokens} = grammar.tokenizeLine("'I am not \\n escaped'")
+    expect(tokens[0]).toEqual value: "'", scopes: ["source.toml", "string.quoted.single.toml", "punctuation.definition.string.begin.toml"]
+    expect(tokens[1]).toEqual value: 'I am not \\n escaped', scopes: ["source.toml", "string.quoted.single.toml"]
+    expect(tokens[2]).toEqual value: "'", scopes: ["source.toml", "string.quoted.single.toml", "punctuation.definition.string.end.toml"]
+
+  it "tokenizes multiline strings", ->
+    {tokens} = grammar.tokenizeLine('"""\nI am a\\\nstring"""')
```

Comment:
To make it clear that this is multiline you can convert this into a `tokenizeLines` instead.  See https://github.com/atom/language-javascript/blob/master/spec/javascript-spec.coffee#L347 as an example.